reference/filter/filters.xml

```diff
@@ -0,0 +1,774 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- $Revision$ -->
+<!-- EN-Revision: 06e8e34994d0d5183cf43b72932a2ffa48406dce Maintainer: towserchen Status: ready -->
+<chapter xml:id="filter.filters" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
+ <title>Types of filters</title>
+
+ <!--Validate filters: {{{-->
+ <section xml:id="filter.filters.validate">
+  <title>Validate filters</title>
+   <para>
+    <table>
+     <title>Listing of filters for validation</title>
+     <tgroup cols="5">
+      <thead>
+       <row>
+        <entry>ID</entry>
+        <entry>Name</entry>
+        <entry>Options</entry>
+        <entry>Flags</entry>
+        <entry>Description</entry>
+       </row>
+      </thead>
+      <tbody>
+       <row>
+        <entry><constant>FILTER_VALIDATE_BOOLEAN</constant></entry>
+        <entry>"boolean"</entry>
+        <entry>
+         <parameter>default</parameter>
+        </entry>
+        <entry>
+          <constant>FILTER_NULL_ON_FAILURE</constant>
+        </entry>
+        <entry>
+         <para>
+          当值为 "1"、 "true"、 "on" 和 "yes" 时返回 &true;
+          其他情况返回 &false;
```

Comment:
```
          Returns &true; for "1", "true", "on" and "yes".
          Returns &false; otherwise.
```
- 请添加句号
- `"1"、 "true"、 "on" 和 "yes"` “和”这个字把 and 翻译出来了，个人觉得无必要

Reply:
“yes” 和 “and” 阅读时容易搞混，是否把“and”翻译出来更好。